Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -386,7 +385,6 @@ - (BOOL) updateInformationForCurrentTable
 	}
 
 	if (tableEncoding != nil) {
-		[tableEncoding release];
 	}
```

Comment:
Empty if block (remove)

Reply:
removed block, not necessary, there is a setter below that line